pkg/occlient/occlient.go

```diff
@@ -125,9 +125,6 @@ const (
 	// This is required to persist deployed artifacts across supervisord restarts
 	EnvS2IDeploymentBackupDir = "ODO_DEPLOYMENT_BACKUP_DIR"
```

Comment:
This can be removed, as it shouldn't be used anywhere

Reply:
removed

Reply:
This line is still there :-(

Reply:
because of this https://github.com/openshift/odo/pull/1840#issuecomment-514573783 but then I removed it as we upgraded the image